recipe/build.sh

```diff
@@ -109,6 +109,11 @@ cmake ${CMAKE_ARGS} -LAH -G "Ninja"                                       \
     -DWITH_V4L=$V4L                                                       \
     -DWITH_CUDA=0                                                         \
     -DWITH_CUBLAS=0                                                       \
+    -DHDF5_FOUND=1                                                        \
```

Comment:
i feel like you want to set a flag like `CMAKE_CROSSCOMPILING` it doesn't seem to appear in the log.

Reply:
I'm unsure :( please feel free to push something you feel appropriate.

Reply:
I finally got podman working locally to test and may have found something that works?  shrug.